shared/javascripts/context_messenger.js

```diff
@@ -444,13 +444,18 @@ if (Privly === undefined) {
       return;
     }
     if (to === 'CONTENT_SCRIPT') {
-      // Send message to all content scripts
-      safari.application.activeBrowserWindow.tabs.forEach(function (tab) {
-        // Don't message Privly Applications
-        if (tab.url.indexOf('safari-extension') !== 0) {
-          tab.page.dispatchMessage(payload);
-        }
-      });
+      if (this.getContextName() === 'PRIVLY_APPLICATION') {
+        // Send message to the parent content script
+        parent.postMessage(payload, '*');
```

Comment:
I am surprised that works on safari. It would be better to be explicit. I think this is short for `window.parent`?